curriculum/challenges/english/01-responsive-web-design/basic-css/import-a-google-font.english.md

```diff
@@ -34,7 +34,7 @@ tests:
   - text: Your <code>h2</code> element should use the font <code>Lobster</code>.
     testString: assert($("h2").css("font-family").match(/lobster/i));
   - text: You should use an <code>h2</code> CSS selector to change the font.
-    testString: 'assert(/\s*h2\s*\{\s*font-family\:\s*(\"|")?Lobster(\"|")?(.{0,})\s*;\s*\}/gi.test(code));'
+    testString: assert(/\s*h2\s*\{\s*font-family\:\s*(\"|"|\')?Lobster(\"|"|\')?(.{0,})\s*;\s*\}/gi.test(code));
```

Comment:
```suggestion
    testString: assert(/\s*h2\s*\{\s*font-family\:\s*(['"])?Lobster\1\s*;\s*\}/gi.test(code));
```

Reply:
Great suggestion. Will make the changes!